stack/stack-setup-2.yaml

```diff
@@ -512,6 +512,30 @@ ghc:
             content-length: 111330896
             sha1: 057a8af762d0259204cb0b6d758fc6b31f536d46
 
+    # OpenBSD releases are not compatible.
+    # So we need separate blocks for openbsd64-6.0, openbsd64-6.1, with any bindists that are there.
+    # Each block should be `openbsd64-$GhcBuild`, where $GhcBuild on OpenBSD is the major release.
+    # Worse, the `$GhcBuild` is not yet autodetected and should be specified by hand:
+    # https://docs.haskellstack.org/en/stable/yaml_configuration/#ghc-build
+    openbsd64-maj5-min9:
+        8.0.1:
+            url: "https://github.com/commercialhaskell/ghc/releases/download/ghc-8.0.1-release/ghc-8.0.1-x86_64-unknown-openbsd5.9.tar.xz"
+            content-length: 100619184
+            sha1: 7ed707de43456ddc4534f68a17ef58b7099e9a59
+        8.0.2:
+            url: "https://github.com/commercialhaskell/ghc/releases/download/ghc-8.0.2-release/ghc-8.0.2-x86_64-unknown-openbsd.tar.xz"
+            content-length: 100054832
+            sha1: 426b01e6ac72b5e039eb9626649506bc47eb9510
+    openbsd64-current-XXX: #OpenBSD-current versions aren't compatible enough with each other.
+        # This bindist of 7.10.3 has some issues, see https://github.com/commercialhaskell/stack/issues/416#issuecomment-220797513
+        7.10.3:
+            # Originally from https://haskell.sru-systems.com/ghc-7.10.3-openbsd-x86_64-current.tar.bz2
+            # provided by https://github.com/mrijkeboer
+            url: "https://s3.amazonaws.com/download.fpcomplete.com/ghc/ghc-7.10.3-openbsd-x86_64-current.tar.bz2"
+            content-length: 135191335
+            sha1: 35f8b595434fcbfa2cb6fc0e9561a5a4d87382a5
```

Comment:
This block should probably be dropped, but it seems so should be (maybe?) all bindists compiled on current. Not sure though.